apps/manager_app/device_authentication.c

```diff
@@ -116,54 +116,50 @@ typedef enum {
  */
 static pb_size_t get_request_type(const manager_auth_device_request_t *request);
 
+/**
+ * @brief Sends response of type manager_auth_device_response_t to the host. It
+ * internally encodes the response to type manager_result_t
+ *
+ * @param resp The response to be send to the host.
+ */
+static void send_auth_device_response(manager_auth_device_response_t resp);
```

Comment:
let's use references for `manager_auth_device_response_t` intance. it will occupy 132 bytes on each stack.

Reply:
I have fixed this, thanks Ujjwal